runtimes/peregrine/src/lib.rs

```diff
@@ -707,6 +707,21 @@ impl pallet_did_lookup::Config for Runtime {
 	type UniqueLinkingEnabled = ConstBool<false>;
 }
 
+type UniqueLinkingDeployment = pallet_did_lookup::Instance2;
+impl pallet_did_lookup::Config<UniqueLinkingDeployment> for Runtime {
+	type BalanceMigrationManager = ();
+	type Currency = Balances;
+	type Deposit = constants::did_lookup::DidLookupDeposit;
+	type DidIdentifier = DidIdentifier;
+	type EnsureOrigin = did::EnsureDidOrigin<DidIdentifier, AccountId>;
+	type OriginSuccess = did::DidRawOrigin<AccountId, DidIdentifier>;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeHoldReason = RuntimeHoldReason;
+	type UniqueLinkingEnabled = ConstBool<true>;
+	// TODO: Change
+	type WeightInfo = ();
```

Comment:
can we already add some benchmarks?

Reply:
I mean I can copy the weights we already have. Will do that.

Reply:
Copied the weights here: https://github.com/KILTprotocol/kilt-node/pull/784/commits/01b3ce5d8c5392a9ff4ae1ce44bbab02e9683a73. I noticed that the benchmarks are now failing and the logic must be made generic. I'll do that in a separate PR. For now the weights should be fine.